packages/console/src/assets/docs/tutorial/integrate-sdk/react.mdx

```diff
@@ -157,39 +145,41 @@ const App = () => {
   onButtonClick={() => props.onNext(4)}
 >
 
-Execute signOut() methods will redirect users to the Logto sign out page. After a success sign out, all use session data and auth status will be cleared.
+Calling `.signOut()` will clear all the Logto data in memory and localStorage if they exist.
+
+After signing out, it'll be great to redirect user back to your website. Let's add `http://localhost:1234` as the Post Sign-out URI below, and use it as the parameter when calling `.signOut()`.
 
 <UriInputField appId={props.appId} isSingle={!props.isCompact} name="postLogoutRedirectUris" title="Post Sign-out Redirect URI" />
 
-Add the following code to your web app
+### Implement a sign-out button
 
 ```tsx
-import React from 'react';
-import { useLogto } from '@logto/react';
-
-const SignOutButton = () => {
+const SignOut = () => {
   const { signOut } = useLogto();
 
-  return <button onClick={() => signOut(window.location.origin)}>Sign out</button>;
+  return (
+    <button onClick={() => signOut('http://localhost:1234')}>
+      Sign out
+    </button>
+  );
 };
-
-export default SignOutButton;
 ```
 
 </Step>
 
 <Step
-  title="Further Readings"
-  subtitle="3 steps"
+  title="Further readings"
+  subtitle="4 articles"
   index={4}
   activeIndex={props.activeStepIndex}
   buttonText="admin_console.general.done"
   buttonType="primary"
   onButtonClick={props.onComplete}
 >
 
-- [SDK Documentation](https://link-url-here.org)
-- [OIDC Documentation](https://link-url-here.org)
-- [Calling API to fetch accessToken](https://link-url-here.org)
+- [Customize sign-in experience](https://docs.logto.io/docs/recipes/customize-sie)
+- [Enable SMS or email passcode sign-in](https://docs.logto.io/docs/tutorials/get-started/enable-passcode-sign-in)
+- [Enable social sign-in](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in)
+- [Protect your API](https://docs.logto.io/docs/recipes/protect-your-api)
```

Comment:
shall we use `<a target="_blank">` here to avoid in-page redirect?

Reply:
I have replaced the `<a>` element in Guide mdx provider. Links will be opened in new tabs
<img width="638" alt="image" src="https://user-images.githubusercontent.com/12833674/176434345-469da67d-9e7c-413b-b123-bbbdef7859fc.png">